GitUp/Application/AboutPanel.m

```diff
@@ -0,0 +1,33 @@
+//
+//  AboutPanel.m
+//  Application
+//
+//  Created by Dmitry Lobanov on 08.10.2019.
+//
+
+#import "AboutPanel.h"
+#import "NSWindow+Loading.h"
+@interface AboutPanel ()
+@property(nonatomic, weak) IBOutlet NSTextField* versionTextField;
+@property(nonatomic, weak) IBOutlet NSTextField* copyrightTextField;
+@end
+
+@implementation AboutPanel
++ (instancetype)new {
+  return (AboutPanel *)[self loadWindowFromBundleXibWithName:@"AboutPanel" expectedClass:NSPanel.class];
+}
```

Comment:
We shouldn't be overriding new. Override init which means that either calling new or alloc init will do the correct thing.